llarp/handlers/tun.hpp

```diff
@@ -235,6 +235,9 @@ namespace llarp
       std::unordered_map<huint128_t, llarp_time_t> m_IPActivity;
       /// our ip address (host byte order)
       huint128_t m_OurIP;
+      /// our ip for ip6
```

Comment:
should at least be consistent in comments (ip6 vs ipv6 vs v6)

Reply:
updated